third_party/2and3/datetimerange/__init__.pyi

```diff
@@ -0,0 +1,38 @@
+from typing import Any, Optional, Union, Iterable
+import datetime
+from dateutil.relativedelta import relativedelta
+
+class DateTimeRange:
+    NOT_A_TIME_STR: str = ...
+    start_time_format: str = ...
+    end_time_format: str = ...
+    is_output_elapse: bool = ...
+    separator: str = ...
+    def __init__(self, start_datetime: Optional[str], end_datetime: Optional[str], start_time_format: str = ..., end_time_format: str = ...) -> None: ...
+    def __eq__(self, other) -> bool: ...
+    def __ne__(self, other) -> bool: ...
+    def __add__(self, other) -> DateTimeRange: ...
+    def __iadd__(self, other) -> DateTimeRange: ...
+    def __sub__(self, other) -> DateTimeRange: ...
+    def __isub__(self, other) -> DateTimeRange: ...
+    def __contains__(self, x) -> bool: ...
+    @property
+    def start_datetime(self) -> datetime.datetime: ...
+    @property
+    def end_datetime(self) -> datetime.datetime: ...
+    @property
+    def timedelta(self) -> datetime.timedelta: ...
+    def is_set(self) -> bool: ...
+    def validate_time_inversion(self) -> None: ...
+    def is_valid_timerange(self) -> bool: ...
+    def is_intersection(self, x) -> bool: ...
+    def get_start_time_str(self) -> str: ...
+    def get_end_time_str(self) -> str: ...
+    def get_timedelta_second(self) -> float: ...
+    def set_start_datetime(self, value: Union[datetime.datetime, str], timezone: str = ...) -> None: ...
+    def set_end_datetime(self, value: Union[datetime.datetime, str], timezone: str = ...) -> None: ...
```

Comment:
Since the `timezone` arguments default to `None`, they should have `Optional[str]` as arguments since they obviously accept `None` as well.